constants.js

```diff
@@ -1,4 +1,5 @@
-const API_BASE_URL = window.API_BASE_URL || 'https://api.realdevsquad.com';
+const API_BASE_URL =
+  window.API_BASE_URL || 'https://staging-api.realdevsquad.com';
```

Comment:
Why you changed this?

Reply:
> Why you changed this?

While setting up the dashboard frontend I was following as guided by some RDS members, this was one of the things i was told to change. I'm unsure why this change was made

Reply:
> Why you changed this?

reverted back to original